src/main/scala/group/scala/karazin/circe/literal/extras/macros.scala

```diff
@@ -246,15 +309,81 @@ object macros:
         Type.of[T] match
```

Comment:
The same, let's keep the order.
Byte < Short < Int < Long
Float < Double
etc.